client/src/app/components/edit-profile/edit-profile.component.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../../styles.scss";
+.wrapper {
+    max-width: 575px;
+    margin-left: auto;
+    margin-right: auto;
+    .cover-picture {
+        width: calc(100%-10px);
+        height: 350px;
+        background: center;
+        background-image: url(https://i.pinimg.com/originals/d7/0b/19/d70b19d883f6d08caccef02b8479fbbb.png);
+        position: relative;
+
+        .dot {
+            height: 27px;
+            width: 27px;
+            background: rgba(255, 255, 255, 0.85);
+            border-radius: 50%;
+            position: absolute;
+            bottom: -30px;
+            right: 223px;
+            #add-a-photo {
+                top: 0px;
+                left: -1px;
+                position: absolute;
+                font-size: 20px;
+            }
+        }
+        .profile-pic {
+            height: 150px;
+            width: 150px;
+            border-radius: 50%;
+            position: absolute;
+            bottom: -3em;
+            left: 0;
+            right: 0;
+            margin-left: auto;
+            margin-right: auto;
+            border: solid black 0.5px;
+        }
+        .edit-cover-picture {
+            right: 0;
+            bottom: 0;
+            color: black;
+            position: absolute;
+            background: rgba(255, 255, 255, 0.75);
+            margin-bottom: 1em;
+            margin-right: 1em;
+            #add-a-photo {
+                height: 30px;
+            }
+        }
+    }
+    .container {
+        margin-top: 6em;
+        .bio {
+            margin-top: 1em;
+            margin-bottom: 1em;
+        }
+        .mat-flat-button {
+            color: $mt-white;
+            background-color: $mt-coral;
+        }
```

Comment:
Instead of making this specific styling here, why don't we make a class in the forms.scss called .secondary-action-btn and add those styles there. Whats good about this is that we can re-use this class in other areas for consistency